src/pint/models/timing_model.py

```diff
@@ -2534,6 +2534,32 @@ def validate_toas(self, toas):
         if bad_parameters:
             raise MissingTOAs(bad_parameters)
 
+    def find_empty_masks(self, toas, freeze=False):
+        """Find unfrozen mask parameters with no TOAs before trying to fit
+
+        Parameters
+        ----------
+        toas : pint.toa.TOAs
+        freeze : bool, optional
+            Should the parameters with on TOAs be frozen
+
+        Returns
+        -------
+        list
+            Parameters with no TOAs
+        """
+        bad_parameters = []
+        for maskpar in self.get_params_of_type_top("maskParameter"):
+            par = getattr(self, maskpar)
+            if "TNEQ" in str(par.name) or par.frozen:
```

Comment:
Why is there a special case for TNEQ?

Reply:
I don't actually know - I copied it from the `validate_toas()` method